docs/account-portal/direct-links.mdx

```diff
@@ -51,4 +53,14 @@ Specify the fallback redirects under **Account Portal -> Redirects** in the Cler
   alt="Account Portal redirects"
 />
 
-</Steps>
\ No newline at end of file
+</Steps>
+
+## Prefill sign in and sign up fields
+
+In the case of direct links, the values to be used for prefilling the fields for sign in/up can be specified via the following query parameters:
```

Comment:
```suggestion
In the case of direct links, the values to be used for pre-filling the fields for sign-in or sign-up can be specified via the following query parameters:
```

Reply:
you're right I just checked the dictionary!! I apologize, I was following my silly autocorrect.